simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -54,15 +56,42 @@
  */
 public class PushGateway {
 
-  private final String address;
+  // Visible for testing
```

Comment:
I don't know if this comment is necessary. I wanted to test that the URL is actually normalised, so that passing a trailing slash into the constructor accepting a String is okay.